test/spec/services/tag-appender.spec.js

```diff
@@ -10,6 +10,49 @@ describe('Unit testing tagAppender service', function () {
     });
   });
 
+  describe('Loading with requirejs when available and headElement is head', function () {
+    beforeEach(function () {
+      //Restoring the head element makes requirejs come into action
+      module({
+        headElement: window.document.getElementsByTagName('head')[0]
+      });
+    });
+
+    it('should load the javascript files',  inject (function (tagAppender, $window) {
+      var done = false;
+      tagAppender('base/test/mock/mock-lazyloaded-file.js', 'js').then(function () {
+        expect($window.lazyLoadingWorking).toBeTruthy();
```

Comment:
When you know exactly the value of the variable, you should use `toBe(true)`. `toBeTruthy()` is just for those cases that you want to check that something is defined and you don't really know or care about its value.